### PR TITLE
Crm457 429/disbursements

### PR DESCRIPTION
**Pull request by dwhenry**
## Description of change
Add disbursements tab

## Link to relevant ticket

[CRM457-429](https://dsdmoj.atlassian.net/browse/CRM457-429)

## Screenshots of changes (if applicable)
Screenshot 2023-09-20 at 12 09 25

[CRM457-429]: https://dsdmoj.atlassian.net/browse/CRM457-429?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by imtiazAhmd — DISMISSED** [review]

File: app/view_models/v1/disbursement.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    # TODO: calculate this fro events
+    def adjustments
+      0
+    end
+
+    # NOTE: This is currently designed to show values without VAT being included.
+    def table_fields
+      [
+        other_type.to_s.presence || disbursement_type.to_s,
+        number_to_currency(requested, unit: '£'),
```
[the remark this answers is not in the document]
Comment (dwhenry) [reply app/view_models/v1/disbursement.rb:31]: good point... :(

[the remark this answers is not in the document]
Comment (dwhenry) [reply app/view_models/v1/disbursement.rb:31]: done

File: app/view_models/v1/disbursement.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+module V1
+  class Disbursement < BaseViewModel
+    include ActionView::Helpers::NumberHelper
```
Comment (imtiazAhmd) [inline app/view_models/v1/disbursement.rb:3]: Do we need `NumberHelper` if we are using `NumberTo.pounds` from lib ?

Comment (dwhenry) [reply app/view_models/v1/disbursement.rb:3]: :(

**Review by imtiazAhmd — APPROVED** [review]

**Pull request merged**